doc/articles/create-an-app-rider.md

```diff
@@ -7,9 +7,6 @@ uid: Uno.GettingStarted.CreateAnApp.Rider
 > [!NOTE]
```

Comment:
@jeromelaban I will review the changes for this PR in a few minutes (just a heads up so you are not merging the changes too soon)